systemd/manager.go

```diff
@@ -238,6 +238,17 @@ func (m *systemdUnitManager) GetUnitStates(filter pkg.Set) (map[string]*unit.Uni
 		states[name] = us
 	}
 
+	// add Active enter time to UnitState
+	for name, us := range states {
+		prop, err := m.systemd.GetUnitProperty(name, "ActiveEnterTimestamp")
```

Comment:
Is this not available in the properties available in `getUnitState`? making an extra dbus call per unit is pretty expensive...